zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.1.8
+
+- Bump Z-Wave JS Server to 1.0.0

```

Comment:
Should add a header `Breaking change:` above this?

Reply:
It's not a breaking change because older HA versions can't install it.

Reply:
Right, just wondered if we should point this out more in the notes.